rts/Lua/LuaOpenGL.cpp

```diff
@@ -2728,6 +2729,49 @@ int LuaOpenGL::CreateTexture(lua_State* L)
 	return 1;
 }
 
+int LuaOpenGL::ChangeTextureParams(lua_State* L)
+{
+	if (!lua_isstring(L, 1))
+		return 0;
+
+	const std::string& texName = luaL_checkstring(L, 1);
+
+	const LuaTextures& textures = CLuaHandle::GetActiveTextures(L);
+	const LuaTextures::Texture* tex = textures.GetInfo(texName);	
+
+	//LuaTextures::Texture newTex = *tex;
```

Comment:
This needs a non-const accessor in LuaTextures s.t. `tex->target` (and other members) can be changed directly, othewise any unspecified parameters will revert to their defaults.